server/src/main/java/org/spine3/server/stand/StandFunnel.java

```diff
@@ -109,6 +109,7 @@ public Stand getStand() {
          * <p> The value must not be null.
          *
          * <p> If this method is not used, a default value will be used.
+         * // TODO:13-09-16:dmytro.dashenkov: Correct docs. No default value for Stand is used, null value leads to a fail instead.
```

Comment:
Please correct the docs yourselves. That's a nice catch. 
Of course, `Stand` instance must be set.